### ISSUE TITLE
Using in Swift

### ISSUE THREAD

**Issue by caclymer** [Reflejo/GoogleMapsPodspec#4]
How do you include this in a Swift project? It shows a launch error when i try to build and run on a device. Compiles fine but fails to load, says cannot find library.

**Comment by alwold-if** [comment]
You might want to try my fork in #3, which fixed some similar issues I ran into. Also, if you are getting the launch error on the simulator, it would probably help to include output from  `~/Library/Logs/CoreSimulator/CoreSimulator.log` to get the specific error.

**Comment by holiday** [comment]
Thanks for that, `./build.py` works now! However when i drop in the GoogleMaps.framework, it builds fine but then i see this problem as soon as it launches. This seems like the library needs to be added to the embedded binaries? I tried that and i get a code signing issue (testing with SDKDemos project).

```
dyld`dyld_fatal_error:
->  0x120051088 <+0>: brk    #0x3
```

**Comment by alwold-if** [comment]
Are you including it as a cocoapod? That error doesn't give much in the way of info. If you have more errors and/or the full code for the test project, it would probably be easier to tell what's wrong.

**Comment by holiday** [comment]
Yes i am, however ive tried including it as a framework as well with the python script and i keep getting codesigning issues. Take a look at my latest comment here:

https://code.google.com/p/gmaps-api-issues/issues/detail?can=2&start=0&num=100&q=dynamic%20status%3DAccepted&colspec=ID%20Type%20Status%20Introduced%20Fixed%20Summary%20Stars%20ApiType%20Internal&groupby=&sort=&id=9512
